contents/docs/billing/estimating-usage-costs.mdx

```diff
@@ -141,7 +141,8 @@ We don't have data on those yet, unfortunately. In that case, we'd recommend goi
 
 ## How to reduce your PostHog costs
 
-You have two options:
+Each product can be tuned to use only the resources you need. 
 
-- Want to reduce events? Read [How to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
-- Want to reduce recordings? Read [How to only record the sessions you want](/tutorials/limit-session-recordings)
+- For Product analytics, read [how to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
```

Comment:
```suggestion
- For product analytics, read [how to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
```

Reply:
isn't the product name capitalized?

Reply:
"Feature names are in sentence case too"

Reply:
<img width="694" alt="Screen Shot 2023-10-17 at 1 49 02 PM" src="https://github.com/PostHog/posthog.com/assets/34755028/eae5adff-285e-4580-9fb5-3e0f50635e10">

Reply:
huh, ok. ya i usually ignore those boxes 😂